docs/settings/reporting-settings.asciidoc

```diff
@@ -203,26 +203,35 @@ The `file:` protocol is always denied, even if no network policy is configured.
 ==== CSV settings
 
 [[xpack-reporting-csv]] `xpack.reporting.csv.maxSizeBytes` {ess-icon}::
-The maximum {byte-units}[byte size] of a CSV file before being truncated. This setting exists to prevent large exports from causing performance and storage issues. Can be specified as number of bytes. Defaults to `10mb`.
+The maximum {byte-units}[byte size] of a CSV file before being truncated. This setting exists to prevent large exports from causing performance and storage issues. Can be specified as number of bytes. Defaults to `250mb`.
 
 [NOTE]
 ============
-Setting `xpack.reporting.csv.maxSizeBytes` much larger than the default 10 MB limit has the potential to negatively affect the
-performance of {kib} and your {es} cluster. There is no enforced maximum for this setting, but a reasonable maximum value depends
-on multiple factors:
+We recommend using CSV reports to export moderate amounts of data only. The feature enables analysis of data in external tools, but it's not intended for bulk export or to backup {es} data. Setting `xpack.reporting.csv.maxSizeBytes` to larger than 250 MB can negatively affect the performance of your {es} cluster.
 
-* The `http.max_content_length` setting in {es}.
-* Network proxies, which are often configured by default to block large requests with a 413 error.
-* The amount of memory available to the {kib} server, which limits the size of CSV data that must be held temporarily.
+The following deployment configurations may lead to failed report jobs or incomplete reports:
 
-For information about {kib} memory limits, see <<production, using {kib} in a production environment>>.
+* Any shard needed for search is unavailable.
+* Data is stored on slow storage tiers. 
+* Network latency between nodes is high.
+* {ccs-cap} is used.
+
+To export large amounts of data we recommend using {es} APIs directly. See {ref}/point-in-time-api.html[Point in time API], or {ref}/sql-rest-format.html#_csv[SQL with CSV response data format].
 ============
 
 `xpack.reporting.csv.scroll.size`::
 Number of documents retrieved from {es} for each scroll iteration during a CSV export. Defaults to `500`.
+[NOTE]
+============
+If the default size of 500 covers a large number of shards in {es}, you may need to lower this setting. 
```

Comment:
Likewise here, why does the number of shards matter? Sounds like a bug, could you make sure to report it?

Reply:
If the PIT ID gets so long that its size on the wire is a significant concern then that's definitely something to raise with the ES folks. I still don't quite understand how requesting smaller result sets would help you in this case tho, I would expect it would be better to retrieve larger result sets so the whole process takes fewer retrievals so you don't have to transmit the whole PIT ID over the wire so many times.

But how would a user use this guidance in practice? What is a "large number of shards"? 100? 100000? How would they tell how many shards they're targetting anyway? How far should they adjust this setting? Different CSV exports will presumably target different shard counts which suggests this should be something they might want to adjust on each export, rather than a Kibana-wide setting. And pushier users may wonder why this is not something that the system can automatically determine :)

Reply:
I'm going to change this wording since the shard count doesn't apply to the latest iteration of CSV export. Before we used PIT, we used the `_scroll` API, and this phrasing is left over from then.